src/app/interfaces/responses.interfaces.ts

```diff
@@ -0,0 +1,67 @@
+export interface loggedUserInfo {
+  first_name: string;
```

Comment:
this is just object, so we use camelCase

Reply:
I have some questions about it